docs/entity/index.md

```diff
@@ -6,26 +6,26 @@ title: Entity
+| required                                                      | boolean                   | specifies whether the field is required or not.                                                                                 | false         |
+| encrypted                                                     | boolean                   |  encrypts that particular field.                                                                                                | false         |
+| [validators](./validators)                                    | array                     | is used to validate the values of fields using various validators.                                                            | -             |
 
 > [!WARNING]  
-> [Placeholder](https://splunkui.splunkeng.com/Packages/react-ui/Text?section=develop) attribute is deprecated and will be removed in the next major version. Instead, we recommend to use "help" attribute.
+> The [Placeholder](https://splunkui.splunkeng.com/Packages/react-ui/Text?section=develop) attribute is deprecated and will be removed in the next major version. Instead, use the "help" attribute.
 
 ## Common Options
 
 | Property            | Type    | Description                                                                       | Default Value |
 | ------------------- | ------- | --------------------------------------------------------------------------------- | ------------- |
-| placeholder         | string  | `Deprecated` The grey text is shown when the input is empty.                      | -             |
-| display             | boolean | Whether show or hide the field.                                                   | true          |
-| disableonEdit       | boolean | When the form is in edit mode, the field becomes uneditable.                      | false         |
-| enable              | boolean | The enable property sets whether a field is enabled, or not.                      | true          |
-| requiredWhenVisible | boolean | Make field required from UI perspecive when it appears \*used only for visibility | false         |
+| placeholder         | string  | (`Deprecated`) The grey text is shown when the input is empty.                      | -             |
+| display             | boolean | It chooses whether or not to display the field.                                                   | true          |
+| disableonEdit       | boolean | When the form is in edit mode, the field becomes unable to be edited.                    | false         |
+| enable              | boolean | The enable property sets whether a field is enabled or not.                      | true          |
+| requiredWhenVisible | boolean | It displays the required field on the UI when it appears. It is used only for visibility. | false         |
```

Comment:
Via this prop we do not modify display state, we only decide if it should be required when it happens to be visible
```suggestion
| requiredWhenVisible | boolean | It makes the field required on the UI when it appears. It is used only for visibility. | false         |
```